tox.ini

```diff
@@ -0,0 +1,108 @@
+[tox]
```

Comment:
This seems like a big change to our CI. Can you add a big comment block at the top explaining this?

Reply:
I changed the approach, no longer have a root-level tox